project/SparkRedshiftBuild.scala

```diff
@@ -57,6 +57,7 @@ object SparkRedshiftBuild extends Build {
       libraryDependencies ++= Seq(
         "com.amazonaws" % "aws-java-sdk-core" % "1.9.40",
         "com.amazonaws" % "aws-java-sdk-s3" % "1.9.40",
+        "com.amazonaws" % "aws-java-sdk-sts" % "1.9.40" % "test",
```

Comment:
Example code:

```scala
val creds = new InstanceProfileCredentialsProvider().getCredentials.asInstanceOf[AWSSessionCredentials]
STS_ACCESS_KEY_ID = creds.getAWSAccessKeyId
STS_SECRET_ACCESS_KEY = creds.getAWSSecretKey
STS_SESSION_TOKEN = creds.getSessionToken
```

In this case the InstanceProfileCredentialsProvider should wind up creating a background thread to automatically manage refresh of these credentials. So, in a nutshell, it should be _really_ easy for users who use IAM to get this set up: just add this line themselves at the top of their app, then add the configuration to thread these variables into `spark-redshift`'s configuration.

Until we understand all of the complications a little better I don't want to push this into the library itself, but it should be fine to provide this as an example / starter code I think.